holopy/scattering/theory/tmatrix.py

```diff
@@ -29,7 +29,11 @@
 from holopy.scattering.errors import TheoryNotCompatibleError, TmatrixFailure
 from holopy.core.errors import DependencyMissing
 from holopy.scattering.theory.scatteringtheory import ScatteringTheory
-from holopy.scattering.theory.tmatrix_f.S import ampld
+try:
+    from holopy.scattering.theory.tmatrix_f.S import ampld
+    COMPILED_TMATRIX_FORTRAN = True
+except:
```

Comment:
Make this specific to the `ModuleNotFoundError` exception.  Although right now the `import` statement has only one exception, it might change in the future, and it's good practice to catch only specific exceptions when you're passing control back to the program after catching.

Reply:
fixed.